pkg/cmd/cli/cmd/project.go

```diff
@@ -155,6 +155,7 @@ func (o ProjectOptions) RunProject() error {
 
 	contextInUse := ""
 	namespaceInUse := ""
+	isDefaultContext := false
```

Comment:
This is badly named.  I'm trying to figure out what it is.  Is it marking the fact that no matching context was found and the name was treated as a project name?